inspirehep/modules/records/jsonschemas/records/elements/id.json

```diff
@@ -0,0 +1,155 @@
+                        "INSPIRE ID"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "INSPIRE-\\d+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "INSPIRE BAI"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "(\\w+\\.)+\\d+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "ORCID"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "\\d{4}-\\d{4}-\\d{4}-\\d{3}[0-9X]",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "JACOW"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "JACoW-\\d+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "KAKEN"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "\\d+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "ARXIV"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "\\w+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "CERN"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "CCID-\\d+",
+                    "type": "string"
+                }
+            }
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "DESY"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "DESY-\\d+",
```

Comment:
@inspirehep/inspire-content is there a nice regexp that can match DESY user IDs?